apps/mark-scan/backend/src/electrical_testing/background.ts

```diff
@@ -0,0 +1,36 @@
+import { extractErrorMessage, Result, sleep } from '@votingworks/basics';
+
+import { constructAuthMachineState } from '../util/auth';
+import { ServerContext } from './context';
+
+const CARD_READ_INTERVAL_SECONDS = 5;
+
+function resultToString(result: Result<unknown, unknown>): string {
+  return result.isOk()
+    ? 'Success'
+    : `Error: ${extractErrorMessage(result.err())}`;
+}
+
+export async function cardReadLoop({
```

Comment:
Unlike VxScan, we aren't required to test continuous USB drive writing on VxMark as it doesn't continuously export to USB drive